### PR TITLE
update webpacker to v4.0.1

### PR DESCRIPTION
**Pull request by andrewmcodes**
Webpacker 4.0.1 was released yesterday ([link](https://github.com/rails/webpacker/releases/tag/v4.0.1))

This PR updates jumpstart to use the new version of webpacker. Tested and this should work just fine 💯

### CODE REVIEW

File: template.rb [hunk 1]
```diff
@@ -276,7 +276,7 @@ def add_sitemap
   git commit: %Q{ -m 'Initial commit' }
 
   say
-  say "Jumpstart app successfully created! 🙌", :blue
+  say "Jumpstart app successfully created!", :blue
```
Comment (excid3) [inline template.rb]: Don't like my emoji? 😜

**Comment by excid3** [comment]
Thanks @andrewmcodes 👍

File: template.rb (hunk 1 above)
Comment (andrewmcodes) [reply template.rb]: Haha I love it but I was getting an Ascii error!

Comment (excid3) [reply template.rb]: Weird. Webpacker has one in it, so you must have the issue there then too.

**Pull request merged**